IdentityCore/src/oauth2/account/MSIDAccountIdentifier.m

```diff
@@ -23,9 +23,12 @@
 
 #import "MSIDAccountIdentifier.h"
 #import "MSIDClientInfo.h"
+#import "NSString+MSIDExtensions.h"
 
 @implementation MSIDAccountIdentifier
 
+@synthesize hashedHomeAccountId = _hashedHomeAccountId;
```

Comment:
Same here - why is it needed? Can you use readwrite properties instead declared in .m?

Reply:
Wouldn't readwrite allow the value is assigned externally? IMO I think we shouldn't let the hashValue assigned outside, it should be calculated inside the class. But if that's ok, I will update these properties to readwrite.
Using @synthesize is for assigning the ivar value is the only way I know. If I set the property @readonly I can't use _ivar without that synthesize declaration.

Reply:
Addressed as discussed offline.